airgun/entities/syncplan.py

```diff
@@ -57,6 +57,18 @@ def add_product(self, entity_name, products_list):
         view.flash.assert_no_error()
         view.flash.dismiss()
 
+    def remove_product(self, entity_name, products_list):
+        """Remove product from sync plan
+
+        :param str entity_name: sync plan name
+        :param products_list: either one or list of products
+        """
+        view = self.navigate_to(self, 'Edit', entity_name=entity_name)
+        view.products.resources.remove(products_list)
+        assert view.flash.is_displayed
```

Comment:
This is the first time we are seeing the assert statement used here in Airgun PR Review.  @swadeley is your intention is that you wanted to see that error message?

Reply:
Hello @omkarkhatavkar 

I just wanted to make the new function consistent with the one above it: `def add_product`

Reply:
In current master, there are well over 200 `assert`s in Airgun module code:
```
$ ack -c assert airgun/ | cut -d ':' -f 2 | awk '{sum+=$1}END{print sum}'
216
```

So, `assert` is certainly not something exceptional.